tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py{27,35}-django111
+envlist = py{35}-django{22},py{38}-django{22,30},quality
```

Comment:
you can write it as `envlist = py35-django22,py38-django{22,30},quality`, no need for curly brackets when you have only one entity.